### PR TITLE
Update to latest k8s and occm patch

### PR DESCRIPTION
**Pull request by chess-knight**
This PR increases k8s versions to latest patch as well as occm patch versions as discussed in #322.

### CODE REVIEW

File: terraform/files/bin/openstack-kube-versions.inc [hunk 1]
```diff
@@ -3,12 +3,12 @@
 # (c) Kurt Garloff <kurt@garloff.de>, 3/2022
 # SPDX-License-Identifier: Apache-2.0
 # Images from https://minio.services.osism.tech/openstack-k8s-capi-images
-k8s_versions=("v1.18.20" "v1.19.16" "v1.20.12" "v1.21.14" "v1.22.17" "v1.23.15" "v1.24.9" "v1.25.5" "v1.26.0")
+k8s_versions=("v1.18.20" "v1.19.16" "v1.20.15" "v1.21.14" "v1.22.17" "v1.23.16" "v1.24.10" "v1.25.6" "v1.26.1")
```
Comment (garloff) [inline terraform/files/bin/openstack-kube-versions.inc:6]: I did not know that 1.20.x had proceeded that far.
Trouble is that osism does not build and publish node images for these old k8s versions any more.
Maybe we should drop at least 1.18 and 1.19 from this list ...
Anyhow, we should do this with another PR, not here, so this is good!

**Review by garloff — APPROVED** [review]
Thanks, Roman!

File: terraform/files/bin/openstack-kube-versions.inc (hunk 1 above)
Comment (chess-knight) [reply terraform/files/bin/openstack-kube-versions.inc:6]: Thank you.
I know that osism builds only 1.21+  now, but I wanted to have this list "complete".
